File: XCase.Swagger.ProxyGenerator/Generator/RESTApiProxySettings.cs
namespace XCase.REST.ProxyGenerator.Generator
{
    public class RESTApiProxySettings
    {
        public string BaseUrl { get; set; }
        public string ProxyOutputFile { get; set; }
        public string WebApiAssembly { get; set; }
        public string WebApiConfig { get; set; }
        public RESTApiProxySettingsEndPoint[] EndPoints { get; set; }
    }
}

File: platform/src/java/com/xcase/rest/generator/RESTApiProxySettings.java
package com.xcase.rest.generator;

import java.lang.invoke.MethodHandles;
import java.util.*;
import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;

public class RESTApiProxySettings
{
    public String BaseUrl;
    public String ProxyOutputFile;
    public String WebApiAssembly;
    public String WebApiConfig;
    public RESTApiProxySettingsEndpoint[] EndPoints;
}
